app/services/container_dashboard_service.rb

```diff
@@ -175,8 +173,54 @@ def ems_utilization
           :yData => ["used"] + used_mem.values.map { |m| (m / 1024.0).round }
         }
       }
-    else
-      {}
     end
   end
+
+  def hourly_network_trends
+    resource_ids = @ems.present? ? [@ems.id] : ManageIQ::Providers::ContainerManager.all.pluck(:id)
+    hourly_network_trend = Hash.new(0)
+    hourly_metrics =
+      MetricRollup.find_all_by_interval_and_time_range("hourly", 24.hours.ago.beginning_of_hour.utc, Time.now.utc)
+    hourly_metrics =
+      hourly_metrics.where('resource_type = ? AND resource_id in (?)', 'ExtManagementSystem', resource_ids)
```

Comment:
2 changes:

```ruby
resource_ids = @ems.present? @ems.id : ManageIQ::Providers::ContainerManager.all.select(:id)
hourly_metrics.where(:resource_type => 'ExtManagementSystem', :resource_id => resource_ids)
```

If you note, that becomes a sub query. Not sure if you want to bring thise ids back, but I'm guessing not.

Also, is there a way to limit the resource type rather than joining to the ems table?